employees_info/src/main/java/com/teachmeskills/servlet/UserDeleteServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.teachmeskills.servlet;
+
+import com.teachmeskills.util.PostgresDriverManager;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@WebServlet("/delete")
+public class UserDeleteServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {

```

Comment:
It should be performed with doPost method

Reply:
Need more info. Homework says, that we receve parameter in serach string (/delete?id=...). Can't understand, why we need to do it with post method.

Reply:
Yjere is no rxact words about search string. In the requirements it's just mentioned, that this method should retrieve ID.
But according to the logic of using HTTP methods, all modifications should be performed by POST method

Reply:
Fixed.